skills-exercises/intro-video-skills/README.md

```diff
@@ -0,0 +1,28 @@
+# Instructions
+
+## **Submission**
+
+### Put the URL of the loom or youtube video here.
+
+### URL: https://youtu.be/NXLm1jzHavM
+
+_Example-_
+
+URL: https://www.youtube.com/watch?v=dQw4w9WgXcQ
```

Comment:
url was not correct right i guess..

Reply:
Dude, that's the example URL.

Reply:
it is on Line 7
### URL: https://youtu.be/NXLm1jzHavM

https://github.com/iamhimateja/pesto-exercise-starter/pull/3/files#diff-5b2ff515aaf3582fb20166268711e324R7

Reply:
sorry didn't see that ...